### PR TITLE
XED-4293/Changed EDX SGA name

### PR DESCRIPTION
**Pull request by BalaDurgaM**
### Description

Changed edx SGA name

### Jira Card

https://crystaldelta.atlassian.net/browse/XED-4293

### CODE REVIEW

**Review by kishored93 — APPROVED** [review]

File: edx_sga/sga.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ class StaffGradedAssignmentXBlock(StudioEditableXBlockMixin, ShowAnswerXBlockMix
 
     display_name = String(
         display_name=_("Display Name"),
-        default=_('Staff Graded Assignment'),
+        default='Student Assessment',
```
Comment (vinaycd) [inline edx_sga/sga.py]: Use `ugettext_lazy()` /  `_()`

Comment (BalaDurgaM) [reply edx_sga/sga.py]: image
this was the previous commit removed by Ganesh

Comment (vinaycd) [reply edx_sga/sga.py]: https://github.com/mitodl/edx-sga/blob/v0.18.0/edx_sga/sga.py#L91 - Actual repo

Its good to use translation

Comment (BalaDurgaM) [reply edx_sga/sga.py]: Updated the code , vinay please check it

**Review by vinaycd — APPROVED** [review]

**Pull request merged**